### PR TITLE
[#1392] Update dependencies to Jakarta EE9

### PR DESCRIPTION
**Pull request by andymc12**
This updates the dependencies of Jakarta Activation and JAXB to the EE9 level and changes the imports from `javax` to `jakarta` to account for those changes.  Once complete, this will resolve issue #1392 (though we may want to keep it open until final versions of Activation and JAXB are released).

**This PR is still a work in progress.  Please do not merge.**  It builds locally for me because I have built updated versions of dependencies, istack and stax-ex - but until those projects have been released to Maven central (and the version updated in the BOM) it will not build anywhere else.

### CODE REVIEW

**Comment by lukasj** [comment]
@andymc12 here is, probably not so easy to read, dependency graph: https://github.com/eclipse-ee4j/metro-wsit/blob/master/etc/metro-dependencies.png

**Comment by lukasj** [comment]
Following was made available in staging (pass `Pstaging` to the build):

istack-commons 2.0.0-M1
stax-ex 2.0.0-M1

Can you rebase this on current master and integrate these two things, so we could use binaries from this PR for testing updated TCKs, please?

You may also need to exclude dependencies from fastinfoset 1.2.18 temporarily (2.0.0-M1 is not available yet, should be tomorrow)

**Comment by andymc12** [comment]
Yeah, I tried rebasing and so far, no good...  

Can you confirm the istack version?  2.0.0-M1 is lower than the current version (3.0.11).

Thanks

**Comment by lukasj** [comment]
> Can you confirm the istack version? 2.0.0-M1 is lower than the current version (3.0.11).

grrr, my bad (copy&paste), it's 4.0.0-M1

**Comment by lukasj** [comment]
all deps, including fastinfoset 2.0.0-M1, are now in mvn central

**Comment by andymc12** [comment]
Got it - those dependencies work for me.  I just pushed an amended commit after rebasing and using those dependencies.  Thanks!

**Comment by lukasj** [comment]
let's try to make travis happy....
`mvn org.glassfish.copyright:glassfish-copyright-maven-plugin:copyright` will tell which file(s) have wrong copyright (some modified file(s) does not have last year set to `2020`)

File: jaxb-ri/boms/bom/pom.xml [hunk 1]
```diff
@@ -31,13 +31,13 @@
     <description>JAXB Bill of Materials (BOM)</description>
 
     <properties>
-        <jaxb-api.version>2.3.3</jaxb-api.version>
-        <jaxb-api-osgi.version>2.3</jaxb-api-osgi.version>
-        <istack.version>3.0.11</istack.version>
-        <fastinfoset.version>1.2.18</fastinfoset.version>
-        <stax-ex.version>1.8.3</stax-ex.version>
+        <jaxb-api.version>3.0.0-RC1</jaxb-api.version>
```
Comment (lukasj) [inline jaxb-ri/boms/bom/pom.xml]: 3.0.0-RC2

File: jaxb-ri/boms/bom/pom.xml [hunk 2]
```diff
@@ -31,13 +31,13 @@
...
+        <jaxb-api-osgi.version>3.0</jaxb-api-osgi.version>
+        <istack.version>4.0.0-M1</istack.version>
+        <fastinfoset.version>1.2.17</fastinfoset.version>
```
Comment (lukasj) [inline jaxb-ri/boms/bom/pom.xml]: 2.0.0-M1

File: jaxb-ri/boms/bom/pom.xml [hunk 3]
```diff
@@ -31,13 +31,13 @@
...
+        <stax-ex.version>2.0.0-M1</stax-ex.version>
 
-        <jakarta.activation.version>1.2.2</jakarta.activation.version>
+        <jakarta.activation.version>2.0.0-rc1</jakarta.activation.version>
```
Comment (lukasj) [inline jaxb-ri/boms/bom/pom.xml]: 2.0.0-RC2

**Comment by lukasj** [comment]
wrt copywrite... if there are more files to fix, you can use sth like:
`mvn org.glassfish.copyright:glassfish-copyright-maven-plugin:repair -Dcopyright.update=true` to fix all in one go

**Comment by andymc12** [comment]
Thanks for the review and the tip on the copyright tool - that would not have been much fun modifying >300 files' copyrights!  ;-)

I just pushed a new commit with those changes.

**Comment by lukasj** [comment]
so far looks good, the only "missing" part are properly updated osgi related imports, which I hope won't affect TCKs and can be done later. @Tomas-Kraus can you build binary from this PR and use that for your work?

**Pull request merged**